cartridge/ddl-manager.lua

```diff
@@ -0,0 +1,128 @@
+local log = require('log')
+local ddl = require('ddl')
+local yaml = require('yaml')
+local checks = require('checks')
+local errors = require('errors')
+
+local failover = require('cartridge.failover')
```

Comment:
The module should not be inside "cartridge" namespace, I suggest renaming it to `yaml-schema` or something relevant.